common/vendor/ledger-eth.js

```diff
@@ -0,0 +1,284 @@
+/* prettier-ignore */
+/* eslint-ignore */
```

Comment:
We should try to get these in our configs if possible.

Reply:
Linter config was already thrown into `package.json`, so this line is outdated and will be removed. Not yet sure about Prettier though...